.github/ISSUE_TEMPLATE/1-new-plugin-request.yml

```diff
@@ -0,0 +1,37 @@
+name: "✨ New plugin request"
+description: I'd like to request a new plugin.
+title: "New plugin: "
+labels: ["plugin-request"]
```

Comment:
I believe you need to manually create these custom labels first before merging this PR